addons/captives/stringtable.xml

```diff
@@ -176,36 +176,42 @@
             <Polish>Ustawienia więźniów</Polish>
             <Spanish>Ajustes de prisioneros</Spanish>
             <Czech>Nastavení zajatce</Czech>
+            <German>Gefangenen-Einstellungen</German>
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_Description">
             <English>Controls settings for surrender and cable ties</English>
             <Polish>Moduł ten kontroluje ustawienia kapitulacji oraz opasek zaciskowych</Polish>
             <Spanish>Ajustes de control para rendición y precintos</Spanish>
             <Czech>Toto kontroluje nastavení kapitulace a pout</Czech>
+            <German>Einstellungen zur Kapitulation und Kabelbindern</German>
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_handcuffSide_name">
             <English>Can handcuff own side</English>
             <Polish>Skuwanie sojuszników</Polish>
             <Spanish>Se puede  esposar el bando propio</Spanish>
             <Czech>Může spoutat spolubojovníky</Czech>
+            <German>Kann Teamkollegen fesseln</German>
```

Comment:
In welcher Hinsicht ist das Wort Kamerad abwertend? Kameradschaft ist , zumindest in meinen Augen ein sehr positiver Begriff. Ich denke, da wird mir jeder Soldat zustimmen.